Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerSuggestionsViewController_Tests.swift

```diff
@@ -0,0 +1,238 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChatTestTools
+@testable import StreamChat
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerSuggestionsViewController_Tests: XCTestCase {
+    
+    // We need to provide a size to the suggestions view since here we are testing the view in isolation,
+    // and so we can't attach it to a bottomAnchorView. The test to verify the height calculation dependent
+    // on the rows should be done in the parent view controller tests.
+    private let defaultCommandSuggestionsSize = CGSize(width: 360, height: 210)
+    private let defaultMentionSuggestionsSize = CGSize(width: 360, height: 110)
+    
+    // MARK: - Mock Data
+    
+    private let commands: [Command] = [
+        .init(name: "giphy", description: "", set: "", args: "[text]"),
+        .init(name: "ban", description: "", set: "", args: "[@username] [text]"),
+        .init(name: "mute", description: "", set: "", args: "[@username] [text]"),
+        .init(name: "unmute", description: "", set: "", args: "[@username] [text]")
+    ]
```

Comment:
when using real comments, the test will for example fail when we change the giphy icon. I'd suggest creating two test-only commands and add some test images for them to `UIConfig`. The goal is to make the test as focused as possible.